projects/js-packages/components/components/jetpack-footer/index.jsx

```diff
@@ -35,11 +35,12 @@ export default function JetpackFooter( {
 					showText={ false }
 					height="16"
 					className="jp-dashboard-footer__jetpack-symbol"
+					aria-label={ __( 'Jetpack logo', 'jetpack' ) }
 				/>
 				<span className="jp-dashboard-footer__module-name">{ moduleName }</span>
 			</div>
 			<div className="jp-dashboard-footer__footer-right">
-				<a href={ a8cLogoHref }>
+				<a href={ a8cLogoHref } aria-label={ __( 'An Automattic Airline', 'jetpack' ) }>
 					<AutomatticBylineLogo />
```

Comment:
Interesting that `AutomatticBylineLogo` has a `role="img"` attribute while `JetpackLogo` does not. Due to this, things look a little different aria-wise:

<table>
<tr>
	<td>Jetpack Logo
	<td>Automattic Byline
<tr>
	<td><img width="444" alt="Screen Shot 2021-08-19 at 5 03 12 PM" src="https://user-images.githubusercontent.com/4044428/130155152-a44c27ed-1ed1-4175-a9b4-14775cfb6d2b.png">
	<td><img width="444" alt="Screen Shot 2021-08-19 at 5 02 59 PM" src="https://user-images.githubusercontent.com/4044428/130155165-046bdb54-3a2d-4e1b-8d8a-0665c59719eb.png">
</table>

I don't think we need to adjust anything, but thought I'd point it out.